cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -41,7 +41,7 @@
    /*
     * Map containing default values for required configuration properties
     */
-    public static final Map<String, String> CC_DEFAULT_PROPERTIES_MAP;
+    public static final Map<String, Object> CC_DEFAULT_PROPERTIES_MAP;
```

Comment:
why is this object? Won't they all need to be converted to strings anyway when written to the config file?